ctrlutil.go

```diff
@@ -213,7 +213,13 @@ func getLinearControlList(parent Control, fn func(Control) bool) []Control {
 // that has tab-stop feature on. Used by library when processing TAB key
 func NextControl(parent Control, curr Control, next bool) Control {
 	fnTab := func(c Control) bool {
-		return c.TabStop() && c.Visible() && c.Enabled()
+		isVisible := func() bool {
+			parent := c.Parent()
```

Comment:
Nice catch!
Should we loop through all parent's parents to check visibility of all of them up to Window control?

Reply:
Makes sense. I'm also thinking about moving this check to BaseControl's Visible method, what do you think?

Reply:
done.